store/blocks.json

```diff
@@ -50,6 +50,16 @@
       "login-content#default"
     ]
   },
+  "store.orderplaced": {
+    "blocks": [
+      "order-placed"
+    ]
+  },
+  "header": {
+    "blocks": [
+      "logo"
```

Comment:
maybe add a `login` block on the simple header as well

Reply:
The header will need to be customized regardless (like to add a logo) so I think we can leave this up to whoever is customizing the theme.